runtime/build.gradle.kts

```diff
@@ -93,6 +126,14 @@ tasks {
     val generateTestTypes by registering {
         dependsOn(generateKotlinTestTypes, generateJavaTestTypes)
     }
+
+    // DCE is now enable by default in Kotlin 1.3.70/71
+    // and it doesn't work well with commonJS modules
+    // Use of commonJs could be removed since default module is now UMD
+    // but would require some code change
+    val processDceJsKotlinJs by getting {
```

Comment:
See previous related comment

Reply:
After enabling UMD modules I had to set `@JsModule("")` and `@JsNonModule` on Marshaller et Unmarshaller from jsMain and then external on those classes and I would have to change more code. Will leave it this way for now. It can be a future improvement.